src/ucp/core/ucp_mm.inl

```diff
@@ -67,9 +69,10 @@ ucp_memh_put(ucp_context_h context, ucp_mem_h memh, int invalidate)
     if (invalidate) {
         ucs_rcache_region_invalidate(context->rcache, &memh->super,
                 (ucs_rcache_invalidate_comp_func_t)ucs_empty_function, NULL);
+        ucs_rcache_region_put(context->rcache, &memh->super);
```

Comment:
why using the thread-safe version here?

Reply:
it's ucp_mem_unmap() flow, slow path.

Reply:
but it's still protected by same context lock..